README.md

```diff
@@ -111,30 +111,52 @@ T::Coerce[Params].new.from({id: '1'})
 ```
 More examples: [nested params](https://github.com/chanzuckerberg/sorbet-coerce/blob/a56c0c6a363bb49b11e77ac57893afc3d54c6b8c/spec/nested_spec.rb#L18-L26)
 
-## Coercion Error
+## Errors
```

Comment:
Maybe adding a description paragraph here:

There are 3 kinds of errors when the coercion doesn't work successfully. Some of them are configurable to not raise error.